src/data/guilty_gear_xrd_rev_2/dizzy.yml

```diff
@@ -0,0 +1,73 @@
+---
+  name: Dizzy
+  moves:
+    normals:
+      - 5P
+      - 5K
+      - c.S
+      - f.S
+      - 5S
+      - 5D
+      - 4S
+      - 6P
+      - 6H
+      - 2P
+      - 2K
+      - 2S
+      - 2H
+      - 2D
+      - j.P
+      - j.K
+      - j.S
+      - j.H
+      - j.D
+    specials:
+      - name: I used this to catch fish
+        inputs:
+          - 236S
+      - name: For searing cod...
+        inputs:
+          - 236H
+      - name: I used this to pick fruit
+        inputs:
+          - 421S
+      - name: For roasting chestnuts...
+        inputs:
+          - 421H
+      - name: We talked a lot together
+        inputs:
+          - 214P
+          - 214K
+      - name: We fought a lot together
+        inputs:
+          - 214S
+          - 214H
+          - 214D
+      - name: For putting out the light...
+        inputs:
+          - 236P
+      - name: The light was so small in the beginning
+        inputs:
+          - 236K
+      - name: Please, leave me alone
+        inputs:
+          - j.236P
+      - name: What happens when I'm TOO alone # these fucking names
+        inputs:
+          - j.236K
+    overdrives:
+      - name: Imperial Ray
+        inputs:
+          - 63214S
+          - 63214D
+      - name: Don't be overprotective
+        inputs:
+          - 63214P
+      - name: Gamma Ray
+        inputs:
+          - 64641236H
+    instant_kills:
+      - name: Emotional Gamma Ray
+        inputs:
+          - 236236H
+        conditions: in instant kill mode
```

Comment:
Oh, I'd left this out on all other character I've done. I'm okay adding the condition though. Mind adding it for all the others?